inst/shiny/modules/tlg/tlg_option_numeric.R

```diff
@@ -0,0 +1,19 @@
+tlg_option_numeric_ui <- function(id, opt_def, data) {
```

Comment:
suggestion: here and in other similar functions - do you think it would be useful to document the parameters?